app/models/invenio_rdm_record_converter.rb

```diff
@@ -56,17 +56,52 @@ def versions(gf)
 
     def invenio_metadata(gf)
       {
-        "resource_type": resource_type(gf)
+        "resource_type": resource_type(gf.resource_type.shift),
+        "creators": creators(gf.creator)
       }
     end
 
-    def resource_type(gf)
-      irdm_subtype = DH_RESOURCE_TYPES[gf.resource_type.shift]
+    def resource_type(digitalhub_subtype)
+      irdm_subtype = DH_RESOURCE_TYPES[digitalhub_subtype]
       irdm_type = IRDM_RESOURCE_TYPES[irdm_subtype]
 
       {
         "type": irdm_type,
         "subtype": irdm_subtype
       }
     end
+
+    def creators(creators)
+      creators.map do |creator|
+        # dh_user = @known_creators.include?(creator) || (@known_creators << User.find_by(formal_name: creator))
+        # Galter Health Sciences Library is a User on production DigitalHub, but it does NOT have a formal_name value
+        dh_user = User.find_by(formal_name: creator)
+
+        if dh_user.present?
+          creator_type = "personal"
+          dh_user_display_name = dh_user.display_name.split(' ') # split name into components to be reused
+          family_name = dh_user_display_name.pop # remove last value from display name
+          given_name = dh_user_display_name.join(' ') # the remaining strings becomes given name
```

Comment:
I did not see anywhere to modify my display name on DH and my display name had my middle name in it so I think this should be okay to use here. It's easier than using formal name since we can just use `pop` to clean out the family name.